### PR TITLE
DM-5633: Add config and data products

### PR DESCRIPTION
**Pull request by hchiang2**
for running forcedPhotCcd, coaddDriverTask, and multiBandDriverTask

### CODE REVIEW

**Comment by jhoblitt** [comment]
Can one of the admins verify this patch?

File: config/multiBandDriver.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import os.path
+
+from lsst.utils import getPackageDir
+
+for sub in ("mergeCoaddDetections", "measureCoaddSources", "mergeCoaddMeasurements", "forcedPhotCoadd"):
+    path = os.path.join(getPackageDir("obs_decam"), "config", sub + ".py")
+    if os.path.exists(path):
+        getattr(config, sub).load(path)
```
Comment (SimonKrughoff) [inline config/multiBandDriver.py]: This doesn't need to be addressed here, but I noticed that this same code is in the obs_subaru config override.  Does it make sense to push this into the task config?  I'm not sure how to do it, but it always worries me to see this much duplicated code.

Comment (PaulPrice) [inline config/multiBandDriver.py]: This is a feature we need to push up into our framework, somehow, sometime.

Comment (SimonKrughoff) [inline config/multiBandDriver.py]: If there's not a ticket, I've asked Hsin-Fang to file one.

Comment (hchiang2) [inline config/multiBandDriver.py]: I've filed DM-5712 to add that new feature.

**Pull request merged**